lib/services/android-project-service.ts

```diff
@@ -375,7 +376,7 @@ export class AndroidProjectService extends projectServiceBaseLib.PlatformProject
 			const filesForInterpolation = this.$fs.enumerateFilesInDirectorySync(resourcesDestinationDirectoryPath, file => this.$fs.getFsStats(file).isDirectory() || path.extname(file) === constants.XML_FILE_EXTENSION) || [];
 			for (const file of filesForInterpolation) {
 				this.$logger.trace(`Interpolate data for plugin file: ${file}`);
-				await this.$pluginVariablesService.interpolate(pluginData, file, projectData);
+				await this.$pluginVariablesService.interpolate(pluginData, file, projectData, projectData.projectIdentifiers.android);
```

Comment:
why do you pass `projectData.projectIdentifiers.android` as an additional parameter when you already pass `projectData`?

Reply:
Because pluginVariablesService doesn't have the platform context to get the proper identifier for the case.